cli/tests/test_commit_template.rs

```diff
@@ -456,59 +456,13 @@ fn test_log_builtin_templates_colored_debug() {
-    [1m[38;5;14m<<node immutable::◆>>[0m  [1m[38;5;5m<<log change_id shortest prefix::z>>[0m[38;5;8m<<log change_id shortest rest::zzzzzzz>>[39m<<log:: >>[38;5;2m<<log root::root()>>[39m<<log:: >>[1m[38;5;4m<<log commit_id shortest prefix::0>>[0m[38;5;8m<<log commit_id shortest rest::0000000>>[39m<<log::>>
-    [EOF]
-    ");
+    insta::assert_snapshot!(render(r#"builtin_log_oneline"#), @"\u{1b}[1m\u{1b}[38;5;2m<<node working_copy mutable::@>>\u{1b}[0m  \u{1b}[1m\u{1b}[38;5;13m<<log working_copy mutable change_id shortest prefix::r>>\u{1b}[38;5;8m<<log working_copy mutable change_id shortest rest::lvkpnrz>>\u{1b}[39m<<log working_copy mutable:: >>\u{1b}[38;5;9m<<log working_copy mutable email placeholder::(no email set)>>\u{1b}[39m<<log working_copy mutable:: >>\u{1b}[38;5;14m<<log working_copy mutable committer timestamp local format::2001-02-03 08:05:08>>\u{1b}[39m<<log working_copy mutable:: >>\u{1b}[38;5;13m<<log working_copy mutable bookmarks name::my-bookmark>>\u{1b}[39m<<log working_copy mutable:: >>\u{1b}[38;5;12m<<log working_copy mutable commit_id shortest prefix::d>>\u{1b}[38;5;8m<<log working_copy mutable commit_id shortest rest::c315397>>\u{1b}[39m<<log working_copy mutable:: >>\u{1b}[38;5;10m<<log working_copy mutable empty::(empty)>>\u{1b}[39m<<log working_copy mutable:: >>\u{1b}[38;5;10m<<log working_copy mutable empty description placeholder::(no description set)>>\u{1b}[39m<<log working_copy mutable::>>\u{1b}[0m\n<<node mutable::○>>  \u{1b}[1m\u{1b}[38;5;5m<<log mutable change_id shortest prefix::q>>\u{1b}[0m\u{1b}[38;5;8m<<log mutable change_id shortest rest::pvuntsm>>\u{1b}[39m<<log mutable:: >>\u{1b}[38;5;3m<<log mutable author email local::test.user>>\u{1b}[39m<<log mutable:: >>\u{1b}[38;5;6m<<log mutable committer timestamp local format::2001-02-03 08:05:07>>\u{1b}[39m<<log mutable:: >>\u{1b}[1m\u{1b}[38;5;4m<<log mutable commit_id shortest prefix::2>>\u{1b}[0m\u{1b}[38;5;8m<<log mutable commit_id shortest rest::30dd059>>\u{1b}[39m<<log mutable:: >>\u{1b}[38;5;2m<<log mutable empty::(empty)>>\u{1b}[39m<<log mutable:: >>\u{1b}[38;5;2m<<log mutable empty description placeholder::(no description set)>>\u{1b}[39m<<log mutable::>>\n\u{1b}[1m\u{1b}[38;5;14m<<node immutable::◆>>\u{1b}[0m  \u{1b}[1m\u{1b}[38;5;5m<<log immutable change_id shortest prefix::z>>\u{1b}[0m\u{1b}[38;5;8m<<log immutable change_id shortest rest::zzzzzzz>>\u{1b}[39m<<log immutable:: >>\u{1b}[38;5;2m<<log immutable root::root()>>\u{1b}[39m<<log immutable:: >>\u{1b}[1m\u{1b}[38;5;4m<<log immutable commit_id shortest prefix::0>>\u{1b}[0m\u{1b}[38;5;8m<<log immutable commit_id shortest rest::0000000>>\u{1b}[39m<<log immutable::>>\n[EOF]");
 
```

Comment:
nit: these formatting change will be reverted in the next `cargo-insta` release.